backend/src/govsg/middlewares/govsg-verification.middleware.ts

```diff
@@ -0,0 +1,52 @@
+import { Request, Response } from 'express'
+import { GovsgMessage } from '@govsg/models'
+import { GovsgVerification } from '@govsg/models/govsg-verification'
+
+export const listMessages = async (
+  req: Request,
+  res: Response
+): Promise<Response | void> => {
+  const { campaignId } = req.params
+  const { offset, limit } = req.query
+  const { rows, count } = await GovsgMessage.findAndCountAll({
+    where: {
+      campaignId: +campaignId,
+    },
+    offset: +(offset as string),
```

Comment:
Thanks, Kishen!

What sort of defensive checks are you looking at? Is it type checking only or do you have something else in mind? I would see a need for more sanitisation if `campaignId` were directly injected into an SQL statement but in this case, it's a programmatic query.

Let me know what you think!

Reply:
Yup, I was thinking of type checking. Since the query contains some typecasting and an implicit assumption that `campaignId` is a number, I was wondering if it would be better to explicitly check for these before executing the db query. 

Though I suppose either way we'd get an error thrown with the current implementation?

Reply:
Yup, `+` catches that.